docs/distributions/gentoo/installation.md

```diff
@@ -2,10 +2,11 @@
 
 ## Hardware Requirements
 
+* Gentoo Minimal Installation ISO
+    * If you don't have the requirements to use this (see below), then you are required to use another distro's ISO. A recommended distro to use is Ubuntu.
```

Comment:
Could you start the section by explaining how to tell if you should use the gentoo minimal iso or an other distro's iso? Jumping around in the section to figure it out can make it take a bit longer to figure out what one needs to do.

Also, are there instructions or a link on how to use another distro's iso to install gentoo?